From c1490770b8ae5a5147bf773740955d49704df28e Mon Sep 17 00:00:00 2001
From: Li Liu <quic_lliu6@quicinc.com>
Date: Fri, 20 Sep 2024 12:55:04 +0800
Subject: [PATCH 9/9] FROMLIST: arm64: defconfig: Enable SX150X for QCS615 ride
 board

For the QCS615 ride board, enable the SX150X to activate the ANX7625
allowing the DSI to output to the mDP through the external bridge.
The ANX7625 relies on the SX150X chip to perform reset and HPD.

Change-Id: Ia23d463c705a54610ca5b89ac06af2065aee7646
Signed-off-by: Li Liu <quic_lliu6@quicinc.com>
Signed-off-by: Fange Zhang <quic_fangez@quicinc.com>
Link: https://lore.kernel.org/all/20241113-add-display-support-for-qcs615-platform-v2-9-2873eb6fb869@quicinc.com/
Upstream-Status: Submitted [https://lore.kernel.org/all/20241113-add-display-support-for-qcs615-platform-v2-9-2873eb6fb869@quicinc.com/]
---
 arch/arm64/configs/defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 46f5c476c230..cb151bcac077 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -614,6 +614,7 @@ CONFIG_PINCTRL_SM8450_LPASS_LPI=m
 CONFIG_PINCTRL_SC8280XP_LPASS_LPI=m
 CONFIG_PINCTRL_SM8550=y
 CONFIG_PINCTRL_SM8550_LPASS_LPI=m
+CONFIG_PINCTRL_SX150X=y
 CONFIG_PINCTRL_LPASS_LPI=m
 CONFIG_GPIO_ALTERA=m
 CONFIG_GPIO_DAVINCI=y
-- 
2.34.1

